expressEngine/order/service/order.js

```diff
@@ -43,5 +43,22 @@ class Service {
     }
     return { error: error.unableToUpdate };
   }
+
+  get = async ({ query }) => {
+    const { skip, limit } = query;
+    const dataToFind = {
+      collection: Order,
+      skip,
+      limit,
+    };
+    const orders = (await DBService.find(dataToFind));
```

Comment:
I think for fetching orders there should be a filter according to the order status, to get either, placed, canceled or to show all orders